src/style.css

```diff
@@ -0,0 +1,133 @@
+* {
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+body {
+  background: #fff;
+  margin-top: 0;
+  margin-bottom: 0;
```

Comment:
Do you really need it? Look, nothing change 

https://user-images.githubusercontent.com/99403198/214374849-92fc350f-715a-4383-a137-3c7dc24282c0.mov